packages/pyolite-kernel/src/worker.ts

```diff
@@ -78,7 +78,7 @@ export class PyoliteRemoteKernel {
   protected async initKernel(options: IPyoliteWorkerKernel.IOptions): Promise<void> {
     // from this point forward, only use piplite (but not %pip)
     await this._pyodide.runPythonAsync(`
-      await piplite.install(['matplotlib', 'ipykernel'], keep_going=True);
+      await piplite.install(['pillow', 'ipykernel'], keep_going=True);
```

Comment:
one patch down, a couple to go: now we do need `piplite.install(['pillow',)` for the _other_ patch to work, unless we can magic that into existence somehow. but this should get kernels starting.